### DISCUSSION TITLE
Copilot Chat deep search in VS 2022

### DISCUSSION THREAD

**Discussion by jochennaumann** [community/community#78133] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I habe installed Copilot Chat in VS 2022. There seems to be no option to use the latest deep search funktionality to use the whole workspace as reference for a question. It works fine with @workspace in VS Code.
Is it not yet integrated into the extension for VS 2022? When will it be available?

**Comment by ayim** [comment, marked as answer]
@jochennaumann correct that it's not yet available.
@rheapatel23 for the ally-oop and additional details on timeline/availability

**Reply by jochennaumann** [reply]
thanks for the info, no reply from @rheapatel23 though....

**Reply by rheapatel23** [reply]
Hey @jochennaumann, sorry for the delay. Thank you for your feedback, our team is looking into this, however, we don't have a timeline for this.
